### PR TITLE
[BREAK] Remove audio preferences

### LINKED ISSUE #14946: Per-User/Room sound notification does not play
### Description:

When setting the per user notification preferences and selecting a different sound, and saving, the sound does not play. It also does not play even if set back to "User Account Preference".

### Steps to reproduce:

1. Update to latest develop docker.
2. Select a user dm conversation.
3. Select notification preferences from the room's menu.
4. Choose a different sound, and save.
5. Try to get messages from contact.
6. No sound will play.

### Expected behavior:
Custom sound selected should play.

### Actual behavior:
No sound plays, even when reset back to User Account Preference. Not sure if this is relevant, but the console shows
```
TypeError: 'name' member of PermissionDescriptor 'microphone' is not a valid value for enumeration PermissionName.
```

image

### Rocket.Chat

| Version                 | 1.3.0-develop                                             |
| ----------------------- | --------------------------------------------------------- |
| Apps Engine Version     | 1.4.2                                                     |
| Database Migration      | 147                                                       |
| Database Migration Date | Tue Jul 09 2019 05:22:24 GMT-0500 (Central Daylight Time) |
| Installed at            | Mon Jan 29 2018 16:02:11 GMT-0600 (Central Standard Time) |
| Uptime                  | 5 hours, 14 seconds                                       |
| Deployment ID           | JAELT9iyADPbKEGbG                                         |
| PID                     | 1                                                         |
| Running Instances       | 1                                                         |
| OpLog                   | Enabled                                                   |

### Commit

| Hash    | da42a954bfee046aebdeff595f9e71af24b376f4                     |
| ------- | ------------------------------------------------------------ |
| Date    | Thu Jul 4 17:40:42 2019 -0300                                |
| Branch  | develop                                                      |
| Tag     | 1.2.0                                                        |
| Author  | Renato Becker                                                |
| Subject | Adds a new setting to prevent Livechat agents from activating service status when office hours are closed. (#14921) |

### PR DESCRIPTION
**Pull request by sampaiodiego**
Closes #14946 

Remove audio preferences and make them tied to desktop notification preferences.

TL;DR: new message sounds will play only if you receive a desktop notification. you'll still be able to chose to not play any sound

- [ ] migration to remove index

### CODE REVIEW

**Comment by rodrigok** [comment]
@sampaiodiego it will close https://github.com/RocketChat/Rocket.Chat/pull/15704 right?

**Comment by sampaiodiego** [comment]
yes @rodrigok .. thanks

**Comment by FixitMad** [comment]
Am I understanding that you are pulling the custom sounds option and only linking them to desktop preferences? So the issues listed under #14946 will not be fixed and instead removed?

**Comment by sampaiodiego** [comment]
right @FixitMad .. but since this change will be a breaking change, we decided to first fix the other issue first..

**Comment by FixitMad** [comment]
@sampaiodiego , Ahh ok. I was confused. Thanks for the update.
